cub/cub/agent/agent_radix_sort_downsweep.cuh

```diff
@@ -472,30 +340,43 @@ struct AgentRadixSortDownsweep
     OffsetT (&relative_bin_offsets)[ITEMS_PER_THREAD],
     int (&ranks)[ITEMS_PER_THREAD],
     OffsetT block_offset,
-    OffsetT valid_items,
-    ::cuda::std::false_type /*is_keys_only*/)
+    OffsetT valid_items)
   {
     ValueT values[ITEMS_PER_THREAD];
 
     __syncthreads();
 
-    LoadValues(values, block_offset, valid_items, bool_constant_v<FULL_TILE>, bool_constant_v<LOAD_WARP_STRIPED>);
+    if constexpr (FULL_TILE)
```

Comment:
Is this the time where we turn the shouty `FULL_TILE` into something like `IsFullTile`